### PR TITLE
add input/output borders files support from python package

### PR DESCRIPTION
**Pull request by necnec**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

During training and prediction, CatBoost splits the range of values of each floating point feature into intervals, and uses these intervals instead of the true values.
Using the same feature borders during training and prediction improves prediction accuracy.
Currently python users can neither save feature borders after training, nor load them during prediction; while this is possible with the command-line CatBoost tool.

Your task is to close this gap.
You need to add input_borders_file and output_borders_file parameters to classes CatBoost* in the CatBoost python package, and pass them via cython to the C++ implementation.

### CODE REVIEW

**Comment by annaveronika** [comment]
@arcadia-devtools Ship it!

**Comment by annaveronika** [comment]
please agree to the cla, see exaples in other pr-s

File: catboost/python-package/catboost/_catboost.pyx [hunk 1]
```diff
@@ -141,7 +146,6 @@ cdef extern from "catboost/libs/options/enums.h":
     cdef EPredictionType EPredictionType_Probability "EPredictionType::Probability"
     cdef EPredictionType EPredictionType_RawFormulaVal "EPredictionType::RawFormulaVal"
 
-
```
Comment (Evgueni-Petrov-aka-espetrov) [inline catboost/python-package/catboost/_catboost.pyx]: Please undo whitespace change

**Comment by arcadia-devtools** [comment]
@annaveronika, internal review request created: 701583

**Comment by annaveronika** [comment]
@necnec Could you please resolve the conflicts?

**Comment by annaveronika** [comment]
@necnec Do you plan to proceed with the pr?

**Comment by Metamorphus** [comment]
@necnec Please, fix the conflict, it's very quick. I can do it for you, if you give me write access or I can create my PR with your code, if you don't mind.

**Comment by kizill** [comment]
@arcadia-devtools Ship it!

**Comment by annaveronika** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
@annaveronika, internal review request created: 724218

**Comment by annaveronika** [comment]
@necnec Thank you very much for contribution!

**Pull request merged**